src/main/java/com/amazonaws/kinesisvideo/producer/ClientInfo.java

```diff
@@ -76,6 +79,7 @@ public ClientInfo(final long createClientTimeout, final long createStreamTimeout
         mAutomaticStreamingFlags = flag;
         mServiceConnectionTimeout = 0L;
         mServiceCompletionTimeout = 0L;
+        mKvsRetryStrategyCallbacks = new DefaultKvsRetryStrategyCallbacks();
```

Comment:
I think we need a new constructor to set the callbacks when someone doesn't want to use the default. Any reason why we wouldn't want to do that?

Reply:
We are not adding the setting of RetryStrategyCallbacks from Java in these PRs, the purpose is to handle null-initializations in the CPP end and to add the members and classes to Java which are missing for congruency purposes.